greenriver/hmis-frontend
# 186358370-fix-session-expiry-across-tabs

## Description

[session expiration doesn't work across browser tabs
](https://www.pivotaltracker.com/story/show/186358370)

I introduced this bug back in #453

Note: I also removed the useLocalStorage hook which isn't used afaict

## Testing notes

1) Sign in. Open a new tab (in the same browser) and sign out in that tab. Return to original tab and should see 'your session has expired'. This was broken prior to this PR

2) Sign in. Exit the browser without signing out. Start the same browser again and sign in (should have no errors). This currently works but this PR changes the mechanism

## Type of change
[//]: # 'remove options that are not relevant'


## Checklist before requesting review
